src/appleDeviceUtils/captureDdgGlobals.js

```diff
@@ -17,7 +17,30 @@ const secretGlobals = {
     JSONparse: window.JSON.parse,
     Arrayfrom: window.Array.from,
     Promise: window.Promise,
-    ObjectDefineProperty: window.Object.defineProperty
+    ObjectDefineProperty: window.Object.defineProperty,
+    capturedWebkitHandlers: {}
 }
 
-export default secretGlobals
+export { ddgGlobals }
```

Comment:
Can we move the export to the bottom of the file and use a unified export statement?

```js
export { ddgGlobals, captureWebkitHandlers }
```